sledgehammer/start-up.sh

```diff
@@ -103,7 +103,7 @@ if ! suse_ver 12; then
 else
     if [ "$BOOTDEV" != "eth0" ]; then
         mv /etc/sysconfig/network/ifcfg-eth0 /etc/sysconfig/network/ifcfg-$BOOTDEV
-        wicked ifup --timeout ${WAIT_FOR_INTERFACES:-60} $BOOTDEV
+        wicked ifup --timeout ${WAIT_FOR_INTERFACES:-120} $BOOTDEV
```

Comment:
Didn't you just recently raise this timeout? Do you have a suspicion why this is taking so long?